_rules/active-focus-shortcut-1e9941.md

```diff
@@ -0,0 +1,255 @@
+---
+id: 1e9941
+name: Printable key shortcut is only active when component has focus
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts for a user interface component are implemented using only printable characters, then they are only available when that component has focus.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] with at least one [keyboard shortcut][] that requires pressing only [printable character][] keys to trigger an action on a [user interface component][].

```

Comment:
I'm wondering whether the applicability should be turned around:
> … with at least one UI component that can be triggered by a character key shortcut

I don't know which one is best…

Reply:
I'll leave it as it is and wait for other reviewers to give their opinion on this

Reply:
Since Keyboard shortcuts are rare and user interface components are plentiful I would advice first looking whether or not there are keyboard shortcuts at all... but seeing how implementation is up to the implementer as long as pragmatically the implementations come to the same conclusion I would accept both ways but have a slight preference for how it is now.

But I would prefer to move user interface component to the expectation anyway. So I am guessing that that would make this question moot.